plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduRecordCursor.java

```diff
@@ -107,26 +92,9 @@ public boolean advanceNextPosition()
         return true;
     }
 
-    private org.apache.kudu.util.Slice getCurrentRowRawData()
-    {
-        if (currentRow != null) {
-            try {
-                return (org.apache.kudu.util.Slice) ROW_DATA_FIELD.get(currentRow);
-            }
-            catch (IllegalAccessException e) {
-                throw new TrinoException(GENERIC_INTERNAL_ERROR, e);
-            }
-        }
-        return null;
-    }
-
     private int getRowLength()
     {
-        org.apache.kudu.util.Slice rawData = getCurrentRowRawData();
```

Comment:
We could also extract this as a separate commit right ?

Reply:
Yes, will extract this into a separate commit

Reply:
Separate PR is here: https://github.com/trinodb/trino/pull/10986